mne/viz/misc.py

```diff
@@ -291,65 +293,80 @@ def plot_source_spectrogram(stcs, freq_bins, tmin=None, tmax=None,
 
 
 def _plot_mri_contours(mri_fname, surfaces, src, orientation='coronal',
-                       slices=None, show=True):
+                       slices=None, show=True, show_indices=False):
     """Plot BEM contours on anatomical slices."""
     import matplotlib.pyplot as plt
     import nibabel as nib
+    # For ease of plotting, we will do everything in voxel coordinates.
+    _check_option('orientation', orientation, ('coronal', 'axial', 'sagittal'))
 
     # plot axes (x, y, z) as data axes (0, 1, 2)
     if orientation == 'coronal':
         x, y, z = 0, 1, 2
     elif orientation == 'axial':
         x, y, z = 2, 0, 1
-    elif orientation == 'sagittal':
+    else:  # orientation == 'sagittal'
         x, y, z = 2, 1, 0
-    else:
-        raise ValueError("Orientation must be 'coronal', 'axial' or "
-                         "'sagittal'. Got %s." % orientation)
 
     # Load the T1 data
     nim = nib.load(mri_fname)
-    data = nim.get_data()
-    try:
-        affine = nim.affine
-    except AttributeError:  # older nibabel
-        affine = nim.get_affine()
-
+    _, vox_mri_t, _, _, _ = _read_mri_info(mri_fname, units='mm')
```

Comment:
@agramfort prefers explicit `_` precisely because you have to have the correct number -- if the API of the private function changes, under some circumstances at least you get a bomb here instead of possibly a silent error